src/test/java/it/gov/pagopa/payhub/activities/activity/paymentsreporting/PaymentsReportingIngestionFlowFileActivityImplTest.java

```diff
@@ -1,61 +1,77 @@
 package it.gov.pagopa.payhub.activities.activity.paymentsreporting;
 
+import it.gov.digitpa.schemas._2011.pagamenti.CtFlussoRiversamento;
 import it.gov.pagopa.payhub.activities.dao.IngestionFlowFileDao;
-import it.gov.pagopa.payhub.activities.dto.paymentsreporting.IngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.IngestionFlowFileDTO;
 import it.gov.pagopa.payhub.activities.dto.reportingflow.PaymentsReportingIngestionFlowFileActivityResult;
+import it.gov.pagopa.payhub.activities.exception.ActivitiesException;
 import it.gov.pagopa.payhub.activities.service.ingestionflow.IngestionFlowFileRetrieverService;
+import it.gov.pagopa.payhub.activities.service.paymentsreporting.FlussoRiversamentoHandler;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
+import java.io.File;
 import java.nio.file.Path;
+import java.util.Collections;
+import java.util.List;
 import java.util.Optional;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.mockito.Mockito.*;
 
 class PaymentsReportingIngestionFlowFileActivityImplTest {
-
 	private IngestionFlowFileDao ingestionFlowFileDao;
 	private IngestionFlowFileRetrieverService ingestionFlowFileRetrieverService;
+	private FlussoRiversamentoHandler flussoRiversamentoHandler;
```

Comment:
let's take advantage of Mockito integration and let's explicit using `Mock` suffix to understand early if a variable should be a mock or not
```suggestion
	@Mock private IngestionFlowFileDao ingestionFlowFileDaoMock;
	@Mock private IngestionFlowFileRetrieverService ingestionFlowFileRetrieverServiceMock;
	@Mock private FlussoRiversamentoHandler flussoRiversamentoHandlerMock;
	
```